Sources/HttpPipeline/ResponseHeader.swift

```diff
@@ -23,11 +96,10 @@ public enum ResponseHeader {
       return ("Content-Type", mediaType.description)
     case let .location(uri):
       return ("Location", uri)
-    case let .setCookie(cookies):
-      let str = cookies.map { (pair: (key: String, value: String)) in
-        pair.key + "=" + pair.value // TODO: escape
-      }.joined(separator: "; ")
-      return ("Set-Cookie", str)
+    case let .setCookie(key: key, value: value, options: options):
+      // TODO: escape
+      let headerValue = (["\(key)=\(value)"] + options.map(get(\.description))).joined(separator: "; ")
```

Comment:
looks like cookie value escaping isn't so hard. it seems to only be spaces, commas, semicolons and a few control characters (\x00 to \x1F plus \x7F). we should tackle that soon